### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]: Jaws and NVDA are not announcing the name of the edit fields in "Run on web app" dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1460]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Create a new java project.
3. Tab to and select Project ->Azure->Run on web app
4. "Run on  web app" dialog will open. 
5. In "Run on web app" dialog , tab to  and select "Create new web app" radio buttons.
6. Tab to Name edit box 
    Create new edit boxes under "Resource group" expand button and "App Service Plan" expand button .
7. Observe the behaviour of Jaws and NVDA .

**Actual Result:**
Jaws and NVDA are not announcing the name of the edit fields in "Run on web app" dialog

**Jaws behaviour:**
Edit
webapp-180205153135
Type in text
Edit
rg-webapp-180205153135
Type in text

**NVDA Behaviour:**
edit
webapp-180208010232
edit
rg-webapp-180208010232
edit
appsp-180208010232

**Expected Result:**
Jaws and NVDA should announce the name of the edit box
Name property should be 
Name 
Create new edit boxes under "Resource group" expand button and "App Service Plan" expand button .

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![run on web app-edit-no name](https://user-images.githubusercontent.com/22956130/36101034-35b80a12-102e-11e8-83e8-d26740b82c26.jpg)
![8fd93389](https://user-images.githubusercontent.com/22956130/36101039-3a9b6970-102e-11e8-800a-235c1f0667d2.jpg)
![5a44c908](https://user-images.githubusercontent.com/22956130/36101041-3d7048d2-102e-11e8-87c5-82725c463133.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.


- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749932
